src/pages/proposal-page.tsx

```diff
@@ -42,6 +55,13 @@ export default function ProposalPage() {
     isLoading: isLoadingVotes,
     refetch: refetchVotes,
   } = useProposalVotes(proposalId)
+  const [submitting, setSubmitting] = useState(false)
+
+  useFormFooter({
+    onNext: undefined,
+    onPrev: undefined,
+    btnText: 'Execute',
+  })
```

Comment:
For some reason this is not rendering correctly and the button text says `Submit`. When developing locally and making changes the button switches between `Submit` and `Execute`. Any ideas what's going on here?

Reply:
is there like a way check that whatever `useFormFooter` is doing has completed before rendering anything? just a rough idea